udm/beginner/party.md

```diff
@@ -59,38 +60,53 @@ Sample JSON Data
       "infoString": "123 Elm Street, Springfield, 12345, USA"
     }
   ],
+  "PartyContactMech": [
```

Comment:
Adding `PartyContactMech` data to connect ContactMech to Party through partyId & contactMechId